### ISSUE TITLE
Spaces at the end of file and directory names break sdfs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [XianglinLi/opendedup#3]
```
A quick demonstration (/backup/current is a mounted sdfs on Debian lenny
with a couple of packages from squeeze):

 hypatia:/backup/current# touch /tmp/test\
 hypatia:/backup/current# mv /tmp/test\  .
 hypatia:/backup/current# mkdir dirtest\
 hypatia:/backup/current# ls -la
 ls: cannot access test: No such file or directory
 ls: cannot access dirtest: No such file or directory
 total 0
 d????????? ? ? ? ?            ? dirtest
 -????????? ? ? ? ?            ? test

```

Original issue reported on code.google.com by `stur...@gmail.com` on 29 Mar 2010 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the find. Will fix in the next release.
```

Original comment by `sam.silv...@gmail.com` on 29 Mar 2010 at 10:04

**Comment by GoogleCodeExporter** [comment]
```
I think this issue has been fixed in 0.8.11
```

Original comment by `sam.silv...@gmail.com` on 30 Mar 2010 at 5:35
- Changed state: **Fixed**
